app/assets/stylesheets/molecules/_sendbar.scss

```diff
@@ -27,12 +28,14 @@
   display: table-cell;
   vertical-align: middle;
   appearance: none;
-  width: 100%;
-  border: 2px solid $color-teal;
+  width: 1%;
```

Comment:
The molecule is based on `searchbar.scss`, where the width is 1% (https://github.com/codeforamerica/clientcomm-rails/blob/7e89fb3b11ae34eccddaa1886604c143059152cc/app/assets/stylesheets/molecules/_searchbar.scss) — I think anything bigger makes the button look wonky.